torchvision/prototype/transforms/functional/_misc.py

```diff
@@ -9,15 +9,19 @@
 normalize_image_tensor = _FT.normalize
 
 
+def normalize_video(video: torch.Tensor, mean: List[float], std: List[float], inplace: bool = False) -> torch.Tensor:
+    return normalize_image_tensor(video, mean, std, inplace=inplace)
+
+
 def normalize(
     inpt: features.TensorImageTypeJIT, mean: List[float], std: List[float], inplace: bool = False
```

Comment:
All these types such as `features.TensorImageTypeJIT` need to be named to make sense.